### DISCUSSION TITLE
Renaming n?

### DISCUSSION THREAD

**Discussion by jarmitage** [tidalcycles/strudel#395] [Polls]
From @felixroos on Discord:

> n and note both change pitch when you're using synths (one of sine sawtooth triangle square or none set, which defaults to triangle). When you're using samples, n will actually select the sample instead of changing the pitch, see https://strudel.tidalcycles.org/learn/samples/#loading-multiple-samples-per-sound .This is understandably confusing, as n plays different roles, and the difference is rather subtle, as it just depends on the value of s. It might make sense to ignore n completely for synths, or reuse it for another purpose (maybe different variants of the same waveform, or number of partials?). I've checked the docs and it is indeed confusing that some examples still use n to control a synth's pitch so that could be changed immediately. TLDR just use note for pitch and n only to select samples..

This led to the question of whether n should be renamed

**Comment by felixroos** [comment]
I am not sure how to vote.. Changing this would also mean breaking tidal compatibility.. was this already discussed in the tidal repo / forum ?

**Reply by jarmitage** [reply]
Or just a case of adding an alias and preferring that in the docs...?

**Reply by yaxu** [reply]
Yes I think it would be good to discuss a bit more.

**Reply by felixroos** [reply]
there is now this PR: https://github.com/tidalcycles/strudel/pull/674
